cilium/cilium.io
# feat: add Hubble brand

This pull request brings branding for Hubble.

Steps to check:

- open [preview](https://deploy-preview-380--cilium.netlify.app/brand/)
- make sure that everything looks and feels as expected
